### ISSUE TITLE
Bulk Insert

### ISSUE THREAD

**Issue by hhslepicka** [NSLS-II/amostra#8]
Allow bulk insert of containers, samples and requests.
The first two will be heavily used when importing data from files.

**Comment by arkilic** [comment]
Is the existing singular entry model slow? It doesn't make sense to do bulk insert for a sample count less than 1k

**Comment by hhslepicka** [comment]
Isn't it better to execute only one call to the server instead of MxN where M is the number of containers and N is the number of samples? It is not slow now but it can always be faster!

**Comment by arkilic** [comment]
True, but the overhead of setting that one connection up is also considerable and wasteful if the request to be handled is not really "bulk". Also, how many clients will connect to this server? Number of calls is significant for high volume applications. Bulk write is also more code == higher maintenance :construction_worker: 
tldr; break this then I will add the requested feature :hammer:

**Comment by arkilic** [comment]
@hhslepicka now that your gui is functional and you have the create_sample_list(), how do you feel about this?

**Comment by arkilic** [comment]
Commit 	7ef2e84 addresses this
